### DISCUSSION TITLE
`<C-n>`/`<C-p>` doesn't work in development branch

### DISCUSSION THREAD

**Discussion by mathmd** [ahrm/sioyek#1234] [Q&A] [answered]
I just switched from the v2.0.0 to the development branch https://github.com/ahrm/sioyek/releases/tag/sioyek3-alpha0. I noticed that the menu navigation using `<C-n>` and `<C-p>` doesn't work any more, so I have to use arrows instead. Do I have to set some preference parameter to bring this back?

**Comment by ahrm** [comment, marked as answer]
See this: https://github.com/ahrm/sioyek/issues/1195.
